enterprise/cloud.oracle/src/org/netbeans/modules/cloud/oracle/steps/DatasourceNameStep.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.netbeans.modules.cloud.oracle.steps;
+
+import org.netbeans.api.progress.ProgressHandle;
+import org.netbeans.modules.cloud.oracle.assets.AbstractStep;
+import org.openide.NotifyDescriptor;
+import org.openide.util.NbBundle;
+
+/**
+ *
+ * @author Jan Horvath
+ */
+@NbBundle.Messages({
+    "DatasourceName=Datasource Name",
+})
+public class DatasourceNameStep extends AbstractStep<String> {
+    private String selected;
+
+    public DatasourceNameStep() {
+    }
+    
+    @Override
+    public void prepare(ProgressHandle h) {
+    }
+
+    @Override
+    public NotifyDescriptor createInput() {
+        return new NotifyDescriptor.InputLine("DEFAULT", Bundle.DatasourceName()); //NOI18N
```

Comment:
Q: what if the user returns to this step: the previously entered value will be lost (not offered in the input line) ?